README.md

```diff
@@ -87,9 +89,11 @@ configuration options [here](docs/options.md). If you are on Windows, check [her
 Install the IronFunctions CLI tool:
 
 ```sh
-curl -sSL http://get.iron.io/fn | sh
+curl -L http://get.iron.io/fn | sh
```

Comment:
See #501, the `-sSL` already has `-L` in it, I'm not sure why this didn't work for you. cc @rdallman